pkg/cmd/target/remove.go

```diff
@@ -120,14 +121,13 @@ func RemoveTargetWorkspaces(ctx context.Context, client *apiclient.APIClient, ta
 		if workspace.Target != target {
 			continue
 		}
-
-		res, err := client.WorkspaceAPI.RemoveWorkspace(ctx, workspace.Id).Execute()
+		err := workspace_cmd.RemoveWorkspace(ctx, client, &workspace, false)
```

Comment:
https://github.com/user-attachments/assets/d5717d61-decc-4dc4-b9d1-4c9f6e1c6cf2

Because the spinner is started and stopped inside the loop, it will always flicker.

I suggest that you start the spinner outside the loop and stop it after we're done.

The same goes for start and stop all.

Reply:
I see your point about flickering. I tried moving the spinner outside the loop, but it had a downside, users would only know abt success or errors after the loop finished. Instead, I've added and used an inline spinner variant. This gives immediate feedback for each workspace. Let me know if this works for you.

Reply:
Great idea! Nice work.